src/axolotl/core/trainers/trainer_builder/rl.py

```
@@ -0,0 +1,220 @@
+        if self.cfg.use_wandb and training_args.run_name == training_args.output_dir:
+            training_args.run_name = (  # pylint: disable=attribute-defined-outside-init
+                None
+            )
+
+        return training_args
+
+    def build(self, total_num_steps):
+        training_args = self.build_training_arguments(total_num_steps)
+        trainer_kwargs = {}
+        if self.cfg.rl is RLType.IPO:
+            if self.cfg.dpo_label_smoothing:
```

Ruff (0.8.2)

Use a single `if` statement instead of nested `if` statements

Combine `if` statements using `and`

(SIM102)